src/dev.ts

```diff
@@ -29,47 +29,40 @@
 //
```

Comment:
Let's delete this too. No commented code should be put under vcs

Reply:
why? we use this kind of files for development/manual testing so it's handy to have parts of the code commented as well